### ISSUE TITLE
Cannot open the Nav Bar on Google Chrome (Android 13)

### ISSUE THREAD

**Issue by ascopes** [badges/shields#9391]
### Are you experiencing an issue with...

shields.io

### 🐞 Description

Opening the navbar on Google Chrome 114.0.5735.196, Android 13; LE2123 Build/TP1A.220905.001 does not work.

The search bar function when viewing on desktop mode also does not work.

This effectively prevents me using anything on the website other than making static badges, whereas previously I was able to generate badges on mobile easily.

https://github.com/badges/shields/assets/73482956/b774827b-2ec5-44c1-a46c-8b557b91f70a

### 🔗 Link to the badge

N/A

### 💡 Possible Solution

N/A

**Comment by uncenter** [comment]
I'm also seeing that... I tested it on desktop and it was broken too for a minute but seemed to resolve... now I'm seeing that some badges aren't working?

**Comment by uncenter** [comment]
<img width="509" alt="image" src="https://github.com/badges/shields/assets/47499684/2574004c-078f-4493-bb32-3235f7576c10">
I'm seeing this on multiple devices now and I assume this is the issue...

**Comment by ascopes** [comment]
seems odd that a SVG would be a script?

**Comment by calebcartwright** [comment]
Not sure if this is related (or just on my machine), but I'm no longer seeing any of the badge builder content on the right; there is no execute button and associated fields

**Comment by uncenter** [comment]
> Not sure if this is related (or just on my machine), but I'm no longer seeing any of the badge builder content on the right; there is no execute button and associated fields

Yep I believe I'm seeing that now too.

**Comment by chris48s** [comment]
OK. This was a real head-scratcher, but I worked out what is happening.

Basically, we had ended up in a state where some of the containers in the cluster were running one version of the application, and some were running another version. This meant that when a browser made a request for `main.1d01d963.js` (for example), that could have either been:

- a valid javascript file or
- a 404 error (badge)

..depending on which container served it. This lead to obvious breakage!

The good news is: I've re-deployed. If you hit shields.io right now, the frontend should work as expected because every container is running the same version of the application.

The bad news is: Re-deploying isn't a permanent fix. As things stand, that issue is going to happen again at some point.

I think further work/discussion on this (how do we prevent getting into this state again) will move to the
:detective: super secret devops place 🤫
as it is a deployment concern, rather than an issue to be solved at the codebase level.

Thanks for the bug report flagging this issue though.
